contracts/FungibleTokenSwitchboard.cdc

```diff
@@ -0,0 +1,151 @@
+import FungibleToken from "./FungibleToken.cdc"
+
+/// FungibleTokenSwitchboard
+///
+/// The contract that allows an account to receive payments in multiple fungible
+/// tokens using a single {FungibleToken.Receiver}
+///
+pub contract FungibleTokenSwitchboard {
+    
+    // Storage and Public Paths
+    pub let SwitchboardStoragePath: StoragePath
+    pub let SwitchboardPublicPath: PublicPath
+    pub let SwitchboardReceiverPublicPath: PublicPath
```

Comment:
I would maybe remove the `Switchboard` from these, so the full name doesn't get to be too much of a mouthful: `FungibleTokenSwitchboard.SwitchboardStoragePath` is a little bit redundant. :)

Reply:
Yeah totally when using those in another contract / tx the readability is awful.